act-rules-format.md

```diff
@@ -160,26 +160,38 @@ Refer to the [Rule Type](#rule-types) section for detailed definitions of the ru
+- [Conformance Requirements](#conformance-requirements)
+- [Secondary Requirements](#secondary-requirements)
 
 Each [=accessibility requirement=] in the mapping <em class="rfc2119">must</em> include the following:
 
 1. either the name, title, identifier or summary of the accessibility requirement, and
 2. the name of the [=accessibility requirements document=], and
 3. a link or reference to the [=accessibility requirements document=] if one exists, and
-4. the conformance level associated with the accessibility requirement, if one exists.
-
+4. the conformance level associated with the accessibility requirement, if one exists, and
+5. whether the requirement is a conformance requirement or a secondary requirement.
 
 ### Outcome Mapping ### {#outcome-mapping}
+For each conformance requirement in the mapping, an ACT Rule <em class="rfc2119">must</em> indicate what the [=outcomes=] of the rule mean for satisfying an accessibility requirement for that [=test subject=]. 
+
+#### Conformance Requirements ### {#conformance-requirements}
+
+When a rule is designed to test a specific accessibility requirement, the accessibility requirement is mapped as a Conformance Requirement when both of the following conditions are true: 
 
-For each accessibility requirement in the mapping, an ACT Rule <em class="rfc2119">must</em> indicate what the [=outcomes=] of the rule mean for satisfying that accessibility requirement for that [=test subject=]. When one or more of the outcomes for a test target is `failed`, the accessibility requirements are <dfn>not satisfied</dfn> for the test subject. When all of the outcomes are `passed` or `inapplicable`, the accessibility requirements could be <dfn>satisfied</dfn>, or <dfn>further testing is needed</dfn>. Rules that can be used to determine if an accessibility requirement is *satisfied* are called <dfn>satisfying tests</dfn>.
+- Failed Outcomes: When one or more of the outcomes for a test subject is `failed`, the accessibility requirement is <dfn>not satisfied</dfn> for the test subject, and 
+- Passed or Inapplicable Outcomes: When all of the outcomes are `passed` or `inapplicable` for a test subject, the accessibility requirement could be <dfn>satisfied</dfn> or <dfn>further testing is needed</dfn> for the test subject.
+ 
+Rules that can be used to determine if an accessibility requirement is *satisfied* are called <dfn>satisfying tests</dfn>.
+
+- Passed: When all of the outcomes are `passed`, the accessibility requirement is <dfn>satisfied</dfn> for the test subject.
```

Comment:
Given that an outcome of `passed` can mean `further testing is needed`, I don't really understand why are we saying that having only `passed` outcomes equals the accessibility requirement being `satisfied`.

Reply:
I understand this as being the "Passed" condition for "satisfying test".
I.e. "when, additionally, this condition is true, the rule is a _satisfying test_".

I agree it's not super clear.